compiler/rustc_mir_build/src/builder/scope.rs

```diff
@@ -1500,6 +1492,14 @@ fn build_scope_drops<'tcx>(
                     unwind_to = unwind_drops.drops[unwind_to].next;
                 }
 
+                // If the operand has been moved, and we are not on an unwind
```

Comment:
We want to do this early skip *after* we've adjusted the `unwind_to` block.

Reply:
At least I think we do -- this isn't realted, though.

Reply:
Yes, I think you're right, that seems to match what happens above. It makes sense that we would do it after.